networkx/utils/backends.py

```diff
@@ -600,13 +621,13 @@ def _call_if_any_backends_installed(self, /, *args, backend=None, **kwargs):
             # Must run with the given backend.
             # `can_run` only used for better log and error messages.
             # Check `mutates_input` for logging, not behavior.
-            blurb = (
+            kwd_blurb = (
```

Comment:
blurb --> msg
```suggestion
            kwd_msg = (
```
Also, it took me a while to figure out `kwd` stands for `keyword` and it is still confusing-- are we talking about the additional kwargs by the backend(like `get_chunks`) or the `backend=` kwarg itself...

Reply:
The message itself says: 
```f"specified with the `backend='{backend_name}'` keyword argument."```
so I was trying to refer to that use of the term keyword. We have many blurbs/messages to keep track of for exceptions and logging. So I was trying to describe which message this refers to. Ideas?